src/main/g8/build.sbt

```diff
@@ -15,7 +15,7 @@ Test / testOptions += Tests.Argument("-oDF")
 Test / logBuffered := false
 
 run / fork := false
-Global / cancelable := false // ctrl-c
+Global / cancelable := true // ctrl-c
```

Comment:
Let me test this before merging. I think the reason why it was there (instead of using default) was that ctrl-c didn't work otherwise. Could have been inheritance from older sbt.

Reply:
Actually, we don't need that setting. By default it's `cancelable := true`.

Reply:
This doesn't work if it's started with `sbt -Dconfig.resource=local1.conf run`. It is stopped but it's not running coordinated shutdown so sbt mess with the sigterm somehow. That was the original reason I added `cancelable := false`.

There is also another problem. If running within sbt, then ctrl-c and then run again. Then `BindException: [/127.0.0.1:2551] Address already in use`. ActorSystem isn't stopped fully in that case.

I don't think we should spend time on this right now. Just sticking with full `sbt -Dconfig.resource=local1.conf run` should be good enough to start with.

Reply:
I didn't start it again to see if the ports were taken. 

I'm fine to leave it as it. So, now I'm understanding it as: we need to disable it otherwise, with the defaults (true), the ActorSystem is not tear down and the port stays bounded.

Reply:
With defaults it will not run coordinated shutdown.

When running within the sbt shell it will never run coordinated shutdown, independent of the this setting.